test/widgets/actions_test.dart

```diff
@@ -182,8 +182,10 @@ void main() {
 
       final pushedRoutes = <Route<dynamic>>[];
       testNavObserver.onPushed = (route, prevRoute) => pushedRoutes.add(route);
-      final account1Route = InboxPage.buildRoute(accountId: account1.id);
-      final account2Route = InboxPage.buildRoute(accountId: account2.id);
+      final account1Route = MaterialAccountWidgetRoute(
+        accountId: account1.id, page: const InboxPageBody());
+      final account2Route = MaterialAccountWidgetRoute(
+        accountId: account2.id, page: const InboxPageBody());
```

Comment:
These pages don't exist in real life. That rather undermines this test, because it's all about how the navigation stack will behave when one account gets logged out.

… In fact I think contemplating what a version of this test would look like that's updated to be realistic after this PR leads to rediscovering the bug that @chrisbobbe mentioned at https://github.com/zulip/zulip-flutter/pull/1076#pullrequestreview-2474679726 and https://github.com/zulip/zulip-flutter/pull/1076#pullrequestreview-2483685141 .

As Chris said in that latter comment (and I agree), that issue can be left as a followup, because it's not a live bug until we make other changes in the future. So I guess this test can just get a TODO comment pointing at that followup issue.

Reply:
bump: should get that TODO comment